pages/style.css

```diff
@@ -0,0 +1,129 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
I suggest that you switch this to scss, since Sass will give us more capabilities, and Next supports it natively.

Reply:
Done! I used a VS Code Extension to compile my Scss into css, but I'd rather have it in the real build.